app/uk/gov/hmrc/gform/summary/SummaryRenderingService.scala

```diff
@@ -159,6 +196,33 @@ class SummaryRenderingService(
     doc.html.replace("£", "&pound;")
   }
 
+  private def addDataToNotificationPdfHTML(
+    html: String,
+    cache: AuthCacheWithForm,
+    pdfFieldIds: List[FormComponentId],
+    pdfHeader: SmartString,
+    pdfFooter: SmartString)(
+    implicit hc: HeaderCarrier,
+    messages: Messages,
+    curLang: LangADT,
+    lise: SmartStringEvaluator) = {
+
+    val doc = Jsoup.parse(html)
+
+    val headerHtml =
+      if (pdfHeader.value().nonEmpty)
+        notification_pdf_header(cache.formTemplate, markDownParser(pdfHeader)).toString
```

Comment:
This doesn't make sense. Why we don't want to display Hmrc logo, if there is empty header markdown?